dotnet/src/AutoGen.Core/Message/TextMessage.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace AutoGen.Core;
 
-public class TextMessage : IMessage, IStreamingMessage
+public class TextMessage : IMessage, IStreamingMessage, ICanGetTextContent
```

Comment:
Nit: `ICanGetTextContent` already inherits `IMessage, IStreamingMessage`, it is a bit redundant. This also applies to lines 54

Reply:
I noticed the same for `ICanGetToolCalls`, `ICanGetToolCalls : IMessage, IStreamingMessage` ,Is it a preferred style to explicitly add the extra interface?

Reply:
@DavidLuong98 Yes, The `ICanGetTextContent` is to enable customized message type to work with `MessageExtension.GetContent()`